src/main/resources/db/changelog/20211123093750-migrate-environment-column.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20211123093750-1" author="ojanus">
+        <sql>
+            INSERT INTO cp_consumer_environments(cp_consumer_id, environment_id, priority)
+            SELECT id, environment_id, 0 FROM cp_consumer
```

Comment:
Running this in Hosted (or any setup that does not enforce the use of environments), would fail with non-null constraint violation because it would try to insert NULL values in the environment_id column (because the original cp_consumer.environment_id values are null for consumers). To make sure this doesn't happen, let's add a `WHERE environment_id IS NOT NULL` at the end of that `SELECT`